Uplift/Views/Home/FitnessCenterView.swift

```diff
@@ -126,6 +145,79 @@ struct FitnessCenterView: View {
         }
         .frame(minWidth: 84, alignment: .leading)
     }
+    private var equipmentSection: some View {
+
+        VStack(spacing: 12) {
+            sectionHeader(text: "EQUIPMENT")
+            ScrollView(.horizontal) {
+                HStack(spacing: 12) {
+                    ForEach(EquipmentType.allEquipmentTypes, id: \.self) { equipmentType in
+                        if eqpmtTypeDict[equipmentType] != 0 {
+                            VStack(alignment: .leading) {
+                                Text(equipmentTypeToString(eqmtType: equipmentType))
+                                    .font(Constants.Fonts.h3)
+                                    .padding()
+                                    .fixedSize(horizontal: true, vertical: false)
+
+                                equipmentTypeCellView(eqmtType: equipmentType)
+                                    .frame(alignment: .leading)
+
+                                Spacer()
+                            }
+                            .frame(width: 247)
+                            .fixedSize(horizontal: true, vertical: false)
+                            .overlay(
+                                RoundedRectangle(cornerRadius: 12)
+                                    .stroke(Constants.Colors.gray01, lineWidth: 1)
+                                    .upliftShadow(Constants.Shadows.smallLight)
+                            )
+                        }
+                    }
+                }
+            }
+            .scrollIndicators(.hidden)
+        }
+        .padding(.vertical, vertPadding)
+    }
+
+    private func equipmentTypeToString(eqmtType: EquipmentType) -> String {
+
+        if eqmtType == EquipmentType.cardio {
+            return "Cardio Machines"
+        } else if eqmtType == EquipmentType.freeWeights {
+            return "Free Weights"
+        } else if eqmtType == EquipmentType.miscellaneous {
+            return "Miscellaneous"
+        } else if eqmtType == EquipmentType.multiCable {
+            return "Multiple Cables"
+        } else if eqmtType == EquipmentType.plateLoaded {
+            return "Plate Loaded Machines"
+        } else if eqmtType == EquipmentType.racksAndBenches {
+            return "Racks & Benches"
+        } else if eqmtType == EquipmentType.selectorized {
+            return "Precor Selectorized Machines"
+        } else {
+            return "Invalid Equipment Type"
+        }
+    }
+
```

Comment:
Since EquipmentType is an Enum, could you switch this if, if else block into a `switch` statement. It'll make the code a lot cleaner